Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Build the manager binary
+FROM golang:1.13 as builder
+WORKDIR /workspace
+# Copy the jsonnet source
+COPY environments/operator/ environments/operator/
+COPY components/ components/
+COPY jsonnetfile.json jsonnetfile.json
+# Build
+RUN GO111MODULE="on" go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+RUN jb install
+RUN GO111MODULE="on" go get github.com/brancz/locutus
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Should we really use this base image for the upstream container?

Reply:
Any reason why not use this image? It is open for use for everybody.
https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image

Reply:
mainly because we don't use it for anything else in the thanos or prometheus ecosystem

Reply:
Done